pyvo/registry/regtap.py

```diff
@@ -708,8 +708,12 @@ def get_interface(self,
         if not candidates:
             raise ValueError(
                 "No matching interface.")
-        if len(candidates) > 1 and not lax:
-            raise ValueError("Multiple matching interfaces found."
+
+        if len(candidates) > 1:
+            if lax:
+                return candidates
```

Comment:
This changes behaviour. Maybe we should consider changing the default of `lax` to false?